python/lsst/sims/utils/AstrometryUtils.py

```diff
@@ -960,6 +995,25 @@ def _observedFromICRS(ra, dec, pm_ra=None, pm_dec=None, parallax=None, v_rad=Non
 
     """
 
+    if isinstance(ra, np.ndarray):
+        fill_value = np.zeros(len(ra))
+    else:
+        fill_value = 0.0
+
+    if pm_ra is None:
+        pm_ra = fill_value
+    if pm_dec is None:
+        pm_dec = fill_value
+    if parallax is None:
+        parallax = fill_value
+    if v_rad is None:
+        v_rad = fill_value
+
+    are_arrays = _validate_inputs([ra, dec, pm_ra, pm_dec, parallax, v_rad],
+                                  ['ra', 'dec', 'pm_ra', 'pm_dec', 'parallax',
+                                  'v_rad'],
+                                  "observedFromICRS")
+
```

Comment:
_validate_inputs also raises a bunch of standard RuntimeErrors if the user passes in inhomogeneous inputs (e.g. numpy arrays of different lengths, a numpy array for but a float for ra, etc.)

In this instance, I am just using it in that capacity.